env/env_encryption.cc

```diff
@@ -362,6 +362,15 @@ void EncryptedWritableFile::PrepareWrite(size_t offset, size_t len,
   file_->PrepareWrite(offset + prefixLength_, len, options, dbg);
 }
 
+void EncryptedWritableFile::SetPreallocationBlockSize(size_t size) {
+  file_->SetPreallocationBlockSize(size);
+}
+
+void EncryptedWritableFile::GetPreallocationStatus(
+    size_t* block_size, size_t* last_allocated_block) {
+  file_->GetPreallocationStatus(block_size, last_allocated_block);
```

Comment:
Aren't these methods already there for "free" from the WritableFileWrapper?

Reply:
`EncryptedWritableFile` is kind of another wrapper here right? Which overrides some functions, but just forward other functions.